espnet2/tasks/asr.py

```diff
@@ -424,12 +437,18 @@ def required_data_names(
 
     @classmethod
     def optional_data_names(
-        cls, train: bool = True, inference: bool = False
+        cls, train: bool = True, inference: bool = False, aux_tasks: list = []
     ) -> Tuple[str, ...]:
         MAX_REFERENCE_NUM = 4
         retval = []
+        
+        if not inference and aux_tasks is not None:
+            retval = aux_tasks
+            
         retval += ["text_spk{}".format(n) for n in range(2, MAX_REFERENCE_NUM + 1)]
```

Comment:
The changes in https://github.com/espnet/espnet/pull/4753 also added a way to use multiple reference files. Should I change my implementation to fit within that framework? Having both will likely cause some mess/confusion.